randnum.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int main()
+{
+	srand(time(0));
+	int r = rand();
+	while (r > 10)
```

Comment:
You could have actually implemented this in a more efficient way with int r = rand() % 10 + 1;